src/transformers/generation_tf_utils.py

```diff
@@ -1288,6 +1290,29 @@ def adjust_logits_during_generation(
         else:
             return logits
 
+    def _validate_model_kwargs(self, model_kwargs: Dict[str, Any]):
```

Comment:
Same as for PyTorch ([here](https://github.com/huggingface/transformers/blob/main/src/transformers/generation_utils.py#L844)), with `self.forward` replaced with `self.call`